src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -480,6 +483,28 @@ class MySimulation : public SimulationHelper<simDim>
     {
         namespace nvfct = PMacc::nvidia::functors;
 
+        typedef typename PMacc::particles::traits::FilterByIdentifier
+        <
+            VectorAllSpecies,
+            momentumPrev1
+        >::type VectorSpeciesWithMementumPrev1;
+
+        /* copy attribute momentum to momentumPrev1 */
```

Comment:
OPTIONAL:
Please add a comment that this copy is **before** the pusher and that copy **and** pusher are **before** the radiation plugin.